docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,664 @@
+  - start the pod by putting it into the kubelet manifest directory
+
+Here are some apiserver flags you may need to set:
+  - `--cloud_provider=`
+  - `--cloud_config=` if cloud provider requires a config file (gce, aws). If so, need to put config file into apiserver docker image or mount through hostDir.
+  - `--address=${MASTER_IP}`.  
+   - or `--address=127.0.0.1` if you want to run a proxy on the master node.
+  - `--cluster_name=$CLUSTER_NAME`
+  - `--service-cluster-ip-range=$SERVICE_CLUSTER_IP_RANGE`
+  - `--etcd_servers=http://127.0.0.1:4001`
+  - `--tls_cert_file=/srv/kubernetes/server.cert` -%}
+  - `--tls_private_key_file=/srv/kubernetes/server.key` -%}
+  - `--admission_control=$RECOMMENDED_LIST`
+    - See [admission controllers](../admission_controllers.md) for recommended arguments.
+  - `--allow_privileged=true`, only if you trust your cluster user to run pods as root.
+ 
+If you are following the firewall-only security approach, then use these arguments:
+  - `--token_auth_file=/dev/null`
+  - `--insecure-bind-address=$MASTER_IP`
+  - `--advertise-address=$MASTER_IP`
+
+If you are using the HTTPS approach, then set:
+  - `--client_ca_file=/srv/kubernetes/ca.crt`
+  - `--token_auth_file=/srv/kubernetes/known_tokens.csv`
+  - `--basic_auth_file=/srv/kubernetes/basic_auth.csv`
+
+*TODO* document proxy-ssh setup.
+
+##### Apiserver pod template
+*TODO*: convert to version v1.
+
+```json
+{
+"apiVersion": "v1beta3",
+"kind": "Pod",
+"metadata": {"name":"kube-apiserver"},
+"spec":{
+"hostNetwork": true,
+"containers":[
+    {
+    "name": "kube-apiserver",
+    "image": "${APISERVER_IMAGE}",
+    "command": [
+                 "/bin/sh",
+                 "-c",
+                 "/usr/local/bin/kube-apiserver $ARGS"
+               ],
+    "livenessProbe": {
+      "httpGet": {
+        "path": "/healthz",
+        "port": 8080
+      },
+      "initialDelaySeconds": 15,
+      "timeoutSeconds": 15
+    },
+    "ports":[
+      { "name": "https",
+        "containerPort": 443,
+        "hostPort": 443},
+      { "name": "local",
+        "containerPort": 8080,
+        "hostPort": 8080}
+        ],
+    "volumeMounts": [
+        { "name": "srvkube",
+        "mountPath": "/srv/kubernetes",
+        "readOnly": true},
+        { "name": "etcssl",
+        "mountPath": "/etc/ssl",
+        "readOnly": true},
+      ]
+    }
+],
+"volumes":[
+  { "name": "srvkube",
+    "hostPath": {
+        "path": "/srv/kubernetes"}
+  },
+  { "name": "etcssl",
+    "hostPath": {
+        "path": "/etc/ssl"}
+  },
+]
+}}
+```
+
+The `/etc/ssl` mount allows the apiserver to find the ssl root certs so it can
+authenticate external services, such as a cloud provider.
+
+The `/srv/kubernetes` mount allows the apiserver to read certs and credentials stored on the
+node disk.
+
+Optionally, you may want to mount `/var/log` as well and redirect output there.
+
+#### Starting Apiserver
+Place the completed pod template into the kubelet config dir
```

Comment:
1. it looks funny  to have
```
block quoted code
```
2. inside and ordered list
3. do you agree?
4. and the numbers are wrong.